RecoHGCal/TICL/python/iterativeTICL_cff.py

```diff
@@ -36,6 +35,18 @@
     ,ticlPFTask
     )
 
+hfnticlLayerTile = ticlLayerTileProducer.clone(
+    detector = 'HFNose'
+)
+
+hfnticlLayerTileTask = cms.Task(hfnticlLayerTile)
+
+hfniterTICLTask = cms.Task(
+    hfnticlLayerTileTask,
+    hfnticlMIPStepTask,
+    hfnticlEMStepTask
```

Comment:
should there also be a HAD iteration for HF nose?

Reply:
Not is a short time scale.
The hadronic showers are not fully contained within the HFnose (only two lamda) and need to link the HF with a much different granularity.